### PR TITLE
Added package.py file for tioga-utils

### PR DESCRIPTION
**Pull request by itopcuoglu**

### CODE REVIEW

File: repos/exawind/packages/tioga-utils/package.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+# LLC (NTESS). Under the terms of Contract DE-NA0003525 with NTESS, the U.S.
+# Government retains certain rights in this software.
+#
+# This software is released under the BSD 3-clause license. See LICENSE file
+# for more details.
+
+from spack import *
+import os
+
+class TiogaUtils(CMakePackage):
```
Comment (psakievich) [inline repos/exawind/packages/tioga-utils/package.py]:
```suggestion
class TiogaUtils(SMCMakeExtension):
```

File: repos/exawind/packages/tioga-utils/package.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2022, National Technology & Engineering Solutions of Sandia,
+# LLC (NTESS). Under the terms of Contract DE-NA0003525 with NTESS, the U.S.
+# Government retains certain rights in this software.
+#
+# This software is released under the BSD 3-clause license. See LICENSE file
+# for more details.
+
+from spack import *
+import os
```
Comment (psakievich) [inline repos/exawind/packages/tioga-utils/package.py:9]:
```suggestion
from spack import *
import os
from smpackages import *
```

**Review by psakievich — COMMENTED** [review]
This looks fine. I would suggest adding the Spack-manager cmake wrapper so things like `compile_commands.json` get moved to the src, and there is a consistent clean up operation.

File: repos/exawind/packages/tioga-utils/package.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+class TiogaUtils(SMCMakeExtension):
+    git = "https://github.com/Exawind/tioga_utils.git"
+
+    version("master", branch="master")
+
+    depends_on("trilinos")
+    depends_on("yaml-cpp")
+    depends_on("tioga")
+ 
+    def cmake_args(self):
```
Comment (jrood-nrel) [inline repos/exawind/packages/tioga-utils/package.py:22]: @psakievich Can this whole `def` be removed then?

File: repos/exawind/packages/tioga-utils/package.py [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+# for more details.
+
+from spack import *
+import os
+from smpackages import *
+
+class TiogaUtils(SMCMakeExtension):
+    git = "https://github.com/Exawind/tioga_utils.git"
+
+    version("master", branch="master")
```
Comment (jrood-nrel) [inline repos/exawind/packages/tioga-utils/package.py]: This should be the exawind branch I believe.

File: repos/exawind/packages/tioga-utils/package.py (hunk 3 above)
Comment (psakievich) [reply repos/exawind/packages/tioga-utils/package.py:22]: Not currently. I thought I had negated the need for it, but looking at the code I think we still need it for now.

File: repos/exawind/packages/tioga-utils/package.py (hunk 4 above)
Comment (itopcuoglu) [reply repos/exawind/packages/tioga-utils/package.py]: I modified this in the updated PR.

File: repos/exawind/packages/tioga-utils/package.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+from smpackages import *
+
+class TiogaUtils(SMCMakeExtension):
+    git = "https://github.com/Exawind/tioga_utils.git"
+
+    version("exawind", branch="exawind")
+
+    depends_on("trilinos")
+    depends_on("yaml-cpp")
+    depends_on("tioga")
```
Comment (ashesh2512) [inline repos/exawind/packages/tioga-utils/package.py]: please make consistent with https://github.com/Exawind/exawind-builder/blob/main/codes/tioga.bash#L39-L47

Comment (itopcuoglu) [reply repos/exawind/packages/tioga-utils/package.py]: Updated the tioga dependency with the nodegid option.

**Comment by itopcuoglu** [comment]
Can we merge this if everything is resolved?

**Review by psakievich — APPROVED** [review]

**Pull request merged**